src/past/translation/__init__.py

```diff
@@ -219,20 +220,20 @@ def detect_python2(source, pathname):
     if source != str(tree)[:-1]:   # remove added newline
         # The above fixers made changes, so we conclude it's Python 2 code
         logger.debug('Detected Python 2 code: {0}'.format(pathname))
-        with open('/tmp/original_code.py', 'w') as f:
+        with open(os.path.join(temppath,'original_code.py'), 'w') as f:
```

Comment:
@mr-c Thanks for the suggestion. I have made appropriate changes. 
And Simultaneous use of `past` module is an interesting use case. Never thought about it. 👍